adal/build.gradle

```diff
@@ -146,12 +146,13 @@ dependencies {
 
     // 'local' flavor dependencies
     localApi(project(":common")) {
-        transitive = false
+        transitive = true
     }
 
     snapshotApi(group: 'com.microsoft.identity', name: 'common', version: '3.2.0', changing: true)
 
     // 'dist' flavor dependencies
+    //TODO: we will have to change transitive to true once common4j is published
```

Comment:
Should we log a warning from gradle here?

Reply:
We could, but I'm going to leave as is for now..I don't think we will be in this state for too long (two weeks probably at most)